semanticcpg/src/test/scala/io/shiftleft/semanticcpg/language/LocationTests.scala

```diff
@@ -11,8 +10,9 @@ class LocationTests extends WordSpec with Matchers {
       int x = foo(param1);
    }"""
 
-  "should return location for method" in {
-    CodeToCpgFixture().buildCpg(code) { cpg =>
+  CodeToCpgFixture(code) { cpg =>
```

Comment:
Nice! I guess this will give us some speedup?

Reply:
yeah, I think so. It was a bit baffled to see that the CPG was calculated once per test. Don't see any downside to calculating it only once.